filament/backend/src/noop/NoopDriver.cpp

```diff
@@ -35,7 +35,7 @@ ShaderModel NoopDriver::getShaderModel() const noexcept {
 #if defined(__ANDROID__) || defined(IOS) || defined(__EMSCRIPTEN__)
     return ShaderModel::GL_ES_30;
```

Comment:
Please update this to `ShaderModel::MOBILE`.

Reply:
done